.github/actions/npm_install/action.yml

```diff
@@ -0,0 +1,47 @@
+name: 'NPM Install with Retries'
+description: 'Run npm install with exponential backoff retries'
+
+inputs:
+    working-directory:
+        description: 'Directory where to run npm install'
+        required: false
+        default: '.'
+    max-attempts:
+        description: 'Maximum number of retry attempts'
+        required: false
+        default: '5'
```

Comment:
Make sure these being strings doesn't mess up math

Reply:
Of course not, I verified by changing the number of attempts and by doing actual math with the values. Strings are just what everything is in bash